### ISSUE TITLE
Sleep Mode Fix for Version 3.92

### ISSUE THREAD

**Issue by dwmccoy** [jimgraygit/Onion#187]
If sleep mode is Hibernation mode in the Settings menu of version 3.92, it does not put the unit to sleep after specifying a hibernation time limit.  I patched version 3.91 to version 3.92 and I performed a clean install of 3.92, and neither would put the unit to sleep.  I have two Miyoo Mini's running 20220419 firmware.  Both units fail to sleep/hibernate using Onion OS 3.92.  Both units using the Stock OS from Miyoo will hibernate/sleep if it's enabled in the Settings Menu of the device.  This would be a duplicate of #184 if sleep mode is hibernation mode.

**Comment by Sichroteph** [comment]
Hibernation is a broken miyoo feature that force turn off your device and loose your progression. Is this a feature asked ?

**Comment by dwmccoy** [comment]
> Hibernation is a broken miyoo feature that force turn off your device and loose your progression. Is this a feature asked ?

If I understand your reply correctly, the sleep mode fix of 3.92 intentionally disables the ability of hibernation because the system will auto hibernate in the middle of game play causing you to lose your progression?  I personally do not use hibernation but the user who posted #184 does.  If the intent is to disable it because it's a broken feature, wouldn't it be best to just hide it from the system menu until the feature is working?  If a feature is displayed in the settings menu, it would be my assumption that it would function as expected.  When it's broken or doesn't work, a lot of times it's the end user who thinks they've done something wrong to make it not work and not realizing it's a broken feature from Miyoo.  Like I stated, I don't use the feature of hibernation, but I'm sure someone out there wants to.  I placed this issue solely on behalf of #184 thinking it fixed the hibernation mode of the device.
